packages/service-library/src/servicelib/enum_utils.py

```diff
@@ -0,0 +1,13 @@
+# NOTE: this is a duplicate, but currently we don't want to enforce a dependency between
+# service-library and models-library
+# should go away and removed when the following case is solved
+# https://github.com/ITISFoundation/osparc-simcore/issues/4013
+
+from enum import Enum, unique
+
+
+@unique
+class StrAutoEnum(str, Enum):
+    @staticmethod
+    def _generate_next_value_(name, start, count, last_values):
```

Comment:
ah this sucks. @pcrespov can't we prevent this? 
if it remains the only one it's maybe ok... but let's keep it as the very only one.

Reply:
We have na issue for this https://github.com/ITISFoundation/osparc-simcore/issues/4013